EiDiA_frontend/src/components/record/filetable/ElementDropTarget.js

```diff
@@ -0,0 +1,39 @@
+import React, {Component} from 'react';
+import {DropTarget} from 'react-dnd';
+import {DragTypes, fileTypes} from "../../../assets/Constants";
+
+
+const calTarget = {
+    canDrop(props) {
+        return props.type === fileTypes.FOLDER;
+    },
+
+    drop(props, monitor, component) {
+        if (monitor.didDrop() || !monitor.canDrop()) return false;
+        return {component};
+    },
+};
+
+function collect(connect, monitor) {
+    return {
+        connectDropTarget: connect.dropTarget(),
+        canDrop: monitor.canDrop(),
+        isOver: monitor.isOver(),
+    };
+}
+
+class ElementDropTarget extends Component {
+    render() {
+        const {connectDropTarget, isOver} = this.props;
+        return connectDropTarget(
+            <div style={{
+                //TODO: better color handling
+                backgroundColor: isOver ? 'rgba(28, 166, 166,0.3)' : ''
```

Comment:
```suggestion
                backgroundColor: isOver && canDrop ? 'rgba(28, 166, 166,0.3)' : ''
```
braucht evl noch mehr changes in der Datei, damit das funktioniert

Reply:
indeed die bessere Lösung...